oidc-wp/openid-connect-generic
# Only load admin CSS when showing settings page

### All Submissions:






### Changes proposed in this Pull Request:



The `styles-admin.css` file currently loads even on the front-end pages of a site. Visitors should not be loading an extra request for admin-related styles, nor should unrelated admin pages.

This PR changes the enqueue of `styles-admin.css` to only happen when loading the settings page. While this CSS currently only affects the logging table, the naming suggests it could be used for any of the admin UI, which is only the settings page.

### How to test the changes in this Pull Request:

1. While logged out, load a page of your site.
2. Open the Network tab of your browser's developer tools.
3. You should not see `styles-admin.css` loading.
4. Now, log in and go to the OpenID settings page.
5. Looking back at the Network tab, you should now see `styles-admin.css` loading.

### Other information:




Note: I wasn't able to get the tests running properly to inspect/add tests, but hopefully this change should be both easy to review and easy to test manually, which may not require automated tests. If not, I can work on troubleshooting the docker-compose error I was getting in trying to run tests.

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

Only load admin CSS when showing settings page.
